Prace_domowe/Praca_domowa6/Krzysztof_Wolny/PD6.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<p>Wyciąganie wniosków na podstawie pojedynczych próbek jest błędne, bo zawsze można powiedzieć że to był przypadek. Lepiej przeprowadzić testy na większej liczbie próbek, obliczyć średnią, odchylenie standardowe itp. A najlepiej to przeprowadzić test t-Studenta na zgodność średnich. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/399/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>